.github/workflows/lint.yml

```diff
@@ -23,6 +23,21 @@ jobs:
 
       - name: Checkout code
         uses: actions/checkout@v3
+        with:
```

Comment:
I do not know enough about cgo right now, but I have used cgo libraries in the past before in other projects, how is it that I never had to setup these CI compilation workarounds?

Reply:
Because `mcl` is complex project it must be build before usage and built c libraries should be linked with `CGO_LDFLAGS`. If you look at https://github.com/alinush/go-mcl his workflow is to manually download `mcl` project to /usr/local, compile it there and than to setup all those 4 environment variables to point to /usr/local/mcl...

Reply:
I have tested [bls-go-binary](https://github.com/herumi/bls-go-binary) which depends also on `mcl` and it does not require any build steps.

Reply:
Yes, it seams that they are using prebuilt library files. I had a problem because github used different glibc than the one i used to built `.a` files. If we use prebuilt libraries it would be a lot less code (submodule will not be required also) but i am not sure it will work on every machine

Reply:
We have decided to use prebuilt `c static library` `*.a` files. Currently he have builds for linux/amd64 and darwin/arm64 os/architecture. Build process is as simple as it was before